spec/controllers/projects_controller_spec.rb

```diff
@@ -1,5 +1,37 @@
 require 'rails_helper'
 
 RSpec.describe ProjectsController, type: :controller do
+  let(:project) { create :project }
 
+  before { sign_in :user, project.user }
+
+  shared_examples 'authentication' do
+    context 'not singed in' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.